ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -30,8 +30,18 @@ if [[ "$REF_ORG" != "$ORG" ]]; then
 fi
 SERVICE_NAME=$(echo "${BASE_OP^^}" | sed 's/\(.*\)-OPERATOR/\1/'| sed 's/-/\_/g')
 
-export IMAGE_TAG_BASE=${REGISTRY}/${ORGANIZATION}/${OPENSTACK_OPERATOR}
-export OPENSTACK_OPERATOR_INDEX=${IMAGE_TAG_BASE}-index:${PR_SHA}
+# Copy base operator code to home directory
+cp -r /go/src/github.com/${ORG}/${BASE_OP}/ ${HOME}
```

Comment:
Just trying to understand do we need this for the rest of the operators if the PR is not for install_yamls?

Reply:
ack, thanks Douglas.

looks good to me to keep it.